db/models.py

```diff
@@ -1 +1,39 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(
+        verbose_name="This field describes what kind of"
+                     " bonus players can get from it.",
```

Comment:
The verbose_name string is split across two lines. Ensure that it is concatenated properly or use a single string to avoid any potential issues.